app/views/schools/index.html.erb

```diff
@@ -1,15 +1,31 @@
 Schools:
 
-<table class="main_table">
+<table class="main_table hoverable">
   <thead>
-    <tr><th>ID</td><th>Name</th><th>Users</th></tr>
+    <tr>
+      <th>ID</td>
+      <th>Name</th>
+      <th>Users</th>
+      <% if policy(School).update? %>
+          <th></th>
+      <% end %>
+      <% if policy(School).destroy? %>
+        <th></th>
+      <% end %>
+    </tr>
   </thead>
   <tbody>
     <% @schools.each do |school| %>
       <tr>
         <td><%= school.id %></td>
         <td><%= link_to school.name, school %></td>
         <td><%= school.users_count %></td>
+        <% if policy(School).update? %>
+          <td><%= link_to 'Edit', edit_school_path(school) if policy(school).update? %></td>
```

Comment:
Note that on the to school edit page the <- back button sends a person to the specific school page with the user list rather than back to the schools page the person came from. The path takes two back buttons to get to where a person starts: [schools list page] -> (edit) -> [school edit page] -> (back) -> [school page] -> (back) -> [schools list page]. I personally don't mind leaving this behaviour if it is too hard to fix though

Reply:
Good point, it should use `:back` instead (430b878)

Reply:
The school edit page already uses `:back`. Not sure what's happening there

Reply:
It's the second parameter that needs to be `:back`. Resolved in 946ebaa.